subsys/dfu/boot/mcuboot.c

```diff
@@ -162,36 +165,89 @@ static int boot_magic_write(u32_t bank_offs)
 	return rc;
 }
 
-static int boot_magic_code_check(const u32_t *magic)
+int boot_read_bank_header(u32_t bank_offset,
```

Comment:
Would you implement intermediary function which could operates on RAW header data plece in RAM - this will be usefully for case when we want to validate incoming image properties prior any data will be  writen to flash?

Reply:
Sure, I'll add a routine which just returns the entire raw structure, with endianness converted.

I'll make it static for now until there's a use for it elsewhere.

Reply:
Done